queries/typst/injections.scm

```diff
@@ -1,2 +1,6 @@
 ((comment) @injection.content
   (#set! injection.language "comment"))
+
+(raw_blck
+  (ident) @injection.language
+  (blob) @injection.content
```

Comment:
Missing a paren here

Reply:
Yeah, my bad.

Reply:
Wrong place ;) We enforce a query format now (you can run `./scripts/format-queries.lua queries/typst`).

Reply:
Thanks for the help!

Reply:
(and please squash)